components/Search/SeparateSearch.tsx

```diff
@@ -0,0 +1,98 @@
+import { useCallback, useEffect, useState } from "react";
+import { unpackSearchResults } from "utils/general";
+import { getEmptyNotice, getSearchResults } from "./utils";
+import { ProductItem } from "./ProductItem";
+import type { SearchResultRecord } from "./types";
+import styles from "./SeparateSearch.module.css";
+
+const SEARCH_THRESHOLD = 1000;
+
+export default function SeparateSearch() {
+  const [savedSearchResults] = useState(() => unpackSearchResults());
```

Comment:
This wasn't picking up the type from the `unpackSearchResults` for me so maybe just adding the type here...
```suggestion
  const [savedSearchResults] = useState<undefined | SearchResultMeta>(() => unpackSearchResults());
```